arcgis-ios-sdk-samples/Scenes/Create terrain surface from a local tile package/README.md

```diff
@@ -0,0 +1,51 @@
+# Create terrain from a local tile package
+
+Set the terrain surface with elevation described by a local tile package.
+
+![Create terrain from a local tile package](create-terrain-from-a-local-tile-package.png)
+
+## Use case
+
+The terrain surface is what the basemap, operational layers, and graphics are draped on. The tile package must be a LERC (limited error raster compression) encoded TPK. Details on creating these are in the [ArcGIS Pro documentation](https://pro.arcgis.com/en/pro-app/help/sharing/overview/tile-package.htm).
+
+Terrain can be loaded offline from tile packages (.tpk).
+
+## How it works
+
+1. Create an `ArcGISScene` and add it to a `SceneView`.
```

Comment:
Cocoa API class names are `AGSScene` and `AGSSceneView`. Here's our API Ref doc for reference - https://developers.arcgis.com/ios/latest/api-reference/